pkg/bpf/bpffs_migrate.go

```diff
@@ -90,35 +92,52 @@ func repinMap(bpffsPath string, name string, spec *ebpf.MapSpec) error {
 	if err != nil {
 		return fmt.Errorf("map not found at path %s: %v", name, err)
 	}
+	defer pinned.Close()
```

Comment:
I assume that closing a pinned map FD doesn't cause any side-effects.

Reply:
Yep, this Close() should've always been there. Without it, the GC closes the fd.